C/common/config_category.cpp

```diff
@@ -146,6 +146,9 @@ ConfigCategory::ConfigCategory(const string& name, const string& json) : m_name(
  */
 ConfigCategory::ConfigCategory(ConfigCategory const& rhs)
 {
+	m_name = string(rhs.m_name);
```

Comment:
I don't think you need to construct the extra string as the string class operator= will do this for you.